tensorflow/core/framework/tensor.cc

```diff
@@ -677,10 +677,10 @@ bool Tensor::CanUseDMA() const {
 #undef CASE
 
 namespace {
-// Recursive print from left dim to right dim
+// Print recursively from left dim to right dim
```

Comment:
Please put punctuation at the end of the comment ("." at the end), as per style guide.